src/main/java/io/bf2/kafka/authorizer/CustomAclAuthorizer.java

```diff
@@ -101,7 +101,8 @@ public class CustomAclAuthorizer implements Authorizer {
     private static final int CREATE_PARTITIONS_APIKEY = 37;
     private static final Logger log = LoggerFactory.getLogger(CustomAclAuthorizer.class);
 
-    static final String CREATE_ACL_INVALID_PRINCIPAL = "Invalid ACL principal name";
+    static final String CREATE_ACL_INVALID_PRINCIPAL_MANAGED = "Invalid ACL principal name: managed principal";
```

Comment:
I'm not sure what `managed principal` would mean to end users. Struggling to think of another term. Maybe `fixed` or `restricted` principal? 

Also would it be better phrased as `Invalid principal name included in ACL`?

Reply:
Restricted sounds good to me. I've also updated the logging to be more consistent.

Included I think is good language since we're dealing with lists.